code/game/dna/genes/goon_disabilities.dm

```diff
@@ -37,9 +37,9 @@
 		block=RADBLOCK
 
 	OnMobLife(var/mob/living/owner)
-		owner.apply_effect(max(owner.radiation, 20), IRRADIATE)
+		owner.apply_effect(max(owner.radiation, 5), IRRADIATE)
```

Comment:
actually this entire PR should just get rid of this max and preserve the 20 value

Because as it is, it does this:
Pick the bigger of 2 values, owner.radiation, or 20
owner.radiation == 0.&nbsp;&nbsp;0 < 20, apply 20 radiation.
owner.radiation == 20.&nbsp;&nbsp;20 == 20, apply 20 radiation.
owner.radiation == 40.&nbsp;&nbsp;40 > 20, apply 40 radiation.
owner.radiation == 80.&nbsp;&nbsp;80 > 20, apply 80 radiation.
owner.radiation == 160.&nbsp;&nbsp;160 > 20, apply 160 radiation.
owner.radiation == 320.&nbsp;&nbsp; Owner is dead.